### DISCUSSION TITLE
What would cause Save View to be greyed out

### DISCUSSION THREAD

**Discussion by dsteinschneider** [rappen/FetchXMLBuilder#533] [Q&A] [answered]
I'm hoping to use FetchXMLBuilder to create a view. I can open existing views but when I clicked save view it didn't add the attributes I added in FetchXMLBuilder . I then tried creating a new Query in the Query Builder but Save View or Save View As are greyed out. Is there a way for me to do this. Thanks

**Comment by rappen** [comment, marked as answer]
Hi!
Since the FXB will not update the `LayoutXML` value, only the `FetchXML` of the view are updated.
Therefore, you can not add other columns, which would make the view not possible to display the view.
You can edit the query, but not the attribute.

You may please add this ass an issue to create this new feature :)

**Comment by dsteinschneider** [comment]
Hi Jonas,

Makes sense, thanks for the quick reply.

Doug
